experimental/cloudstrapper/playbooks/roles/aws-cleanup/tasks/orc8r-efs.yaml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: find all filesystems
+  efs_info:
+  register: valEfs
+
+- name: debug filesystem
+  debug:
+    msg: "{{ valEfs }}" 
+
+- name: delete mount targets, because the efs module doesn't do it on it's own
+  command: aws efs delete-mount-target --mount-target-id "{{ item.mount_target_id }}"
+  with_items:
+    - "{{ valEfs.efs[0].mount_targets }}"
```

Comment:
If there are no entries in efs, this may fail the playbook. Check length prior to deleting mount target IDs

Reply:
Added code that verifies if an entry exists before attempting to remove it